File: batches/batches/batch_06_loop_type_change/ref.py
"""
Reference - Original code for Batch 6: Loop Type Change (NOT cheating)
Sum numbers using for-loop.
"""


def sum_list(nums):
    total = 0
    for n in nums:
        total += n
    return total


if __name__ == "__main__":
    print(sum_list([1, 2, 3, 4, 5]))

File: batches_csharp/batch_06_loop_type_change/ref.cs
using System;

class SumList {
    static int Sum(int[] nums) {
        int total = 0;
        foreach (int n in nums) total += n;
        return total;
    }

    static void Main() {
        int[] nums = {1, 2, 3, 4, 5};
        Console.WriteLine(Sum(nums));
    }
}
